src/TableRow.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import moment from "moment";
+
+function TableRow(props) {
+  const [highlight, setHighlight] = useState(false);
+
+  function highlightRow() {
+    setHighlight(!highlight);
+  }
+
+  function handleShowProfile(event) {
+    props.selectCustomer(props.result.id);
+    event.stopPropagation();
+  }
+
+  return (
+    <tr className={highlight ? "highlighted" : " "} onClick={highlightRow}>
```

Comment:
You've added functionality to highlight a row, when I click on a row, it does not highlight. Can you debug this and explain why?